rtmp-client/src/main/java/net/butterflytv/rtmp_client/RtmpClient.java

```diff
@@ -13,8 +13,18 @@
 
 
     private final static int OPEN_SUCCESS = 1;
+    private final static int TIMEOUT = 10;
     private long rtmpPointer = 0;
 
+    /* Timeout value in seconds */
+    private int timeout = TIMEOUT;
+    /* Sets the timeout variable */
+    public void setTimeout(int timeout) {
+        if (timeout > 0) {
```

Comment:
If it's considered an error to pass a non-positive values, then this should IMO throw IllegalArgumentException rather than silently doing nothing, if such a value is passed.

Alternatively, if it's possible for passing a non-positive value to return the client to whatever its default is, rather than doing nothing, then that would also be fine.

Reply:
Zero or negative values will now reset the timeout to default values.